webui/react/src/hooks/useModal/Experiment/useModalExperimentMove.tsx

```diff
@@ -43,6 +43,7 @@ export interface Settings {
 }
 
 export const settingsConfig: SettingsConfig = {
+  applicableRoutespace: '/experiments',
```

Comment:
What's the purpose of this change?

Reply:
specifying `applicableRoutespace` in general prevents [errors caused by settings interference](https://determinedai.atlassian.net/browse/WEB-490)-- `destroyInactivePane` by itself is not sufficient, at least with current implementation--  and imo is good practice to do whenever possible.

this modal is accessible from ProjectDetails -> Experiments tab and also the experiment details.

before this PR, the former existed at `/projects/{id}/`, but now it exists at `projects/{id}/experiments`, so now both pages have `/experiments` in their routes and it is practicable to specify.

Reply:
if you say this `applicableRoutespace` is opaque and prone to bugs, i would agree with you